addons/web/static/src/views/kanban/kanban_dashboard.scss

```diff
@@ -98,85 +98,6 @@
     }
 }
 
-.o-dropdown--kanban-record-menu {
```

Comment:
this must be kept here for retro compatibility. However, you can scope it to legacy kanban record